website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -55,6 +55,8 @@ Only resources in the `default` admin partition will be replicated to secondary
 
 Client agents will be configured to operate within a specific admin partition. The DNS interface will only return results for the admin partition within the scope of the client.
 
+-> **Consul Dataplane and DNS**: As of Consul v1.14, The Consul clients are optional in Kubernetes since the introduction of [Consul Dataplane](/consul/docs/connect/dataplane), however, to utilize the Consul DNS for service discovery a Consul client is required in the client admin partitions
```

Comment:
```suggestion
-> **Consul Dataplane and DNS:** As of Consul v1.14, Consul clients are optional in Kubernetes since the introduction of [Consul Dataplane](/consul/docs/connect/dataplane). However, a Consul client is required in the client admin partitions to use Consul DNS for service discovery.
```